Mainnet/StandardToken/StandardToken.Tests/StandardTokenTests.cs

```diff
@@ -580,18 +597,19 @@ public void TransferFrom_Self()
         }
 
         [Fact]
-        public void Constructor_Sets_Name_And_Symbol()
+        public void Constructor_Sets_Name_Symbol_And_Decimals()
```

Comment:
Can you update this test for having all parameters ? Currently only TotalSupply is missing. Also give it a generic name `Constructor_Sets_Parameters` after that you can remove `Constructor_Sets_TotalSupply`